torch/nn/parameter.py

```diff
@@ -25,3 +25,8 @@ def __new__(cls, data=None, requires_grad=True):
 
     def __repr__(self):
         return 'Parameter containing:\n' + super(Parameter, self).__repr__()
+
+    def __reduce_ex__(self, proto):
+        tensor = torch._utils._rebuild_tensor_v2(self.storage(), self.storage_offset(), tuple(self.size()),
+                                                 self.stride(), self.requires_grad, self._backward_hooks)
+        return (Parameter, (tensor, self.requires_grad))
```

Comment:
wait, this can't be the correct fix right? You are making an extra `Tensor` instance. Shouldn't `Parameter, (super(Parameter, self), self.requires_grad)` be used instead?